ai-engineering/create.mdx

```diff
@@ -1,133 +1,159 @@
 ---
 title: "Create"
-description: "Learn how to create and define AI capabilities using structured prompts and typed arguments with Axiom."
-keywords: ["ai engineering", "AI engineering", "create", "prompt", "template", "schema"]
+description: "Build AI capabilities using any framework, with best support for TypeScript-based tools."
+keywords: ["ai engineering", "create", "prompt", "capability", "vercel ai sdk"]
 ---
 
-import { Badge } from "/snippets/badge.jsx"
 import { definitions } from '/snippets/definitions.mdx'
 
-The **Create** stage is about defining a new AI <Tooltip tip={definitions.Capability}>capability</Tooltip> as a structured, version-able asset in your codebase. The goal is to move away from scattered, hard-coded string prompts and toward a more disciplined and organized approach to prompt engineering.
+Building an AI <Tooltip tip={definitions.Capability}>capability</Tooltip> starts with prototyping. You can use whichever framework you prefer. Axiom is focused on helping you evaluate and observe your capabilities rather than prescribing how to build them.
+
+TypeScript-based frameworks like Vercel’s [AI SDK](https://sdk.vercel.ai) do integrate most seamlessly with Axiom’s tooling today, but that’s likely to evolve over time.
+
+## Build your capability
+
+Define your capability using your framework of choice. Here’s an example using Vercel's [AI SDK](https://ai-sdk.dev/), which includes [many examples](https://sdk.vercel.ai/examples) covering different capability design patterns. Popular alternatives like [Mastra](https://mastra.ai) also exist.
+
+```ts src/lib/capabilities/classify-ticket.ts expandable
```

Comment:
aside: i HATE that we're collapsing code blocks by default. if we want a code block to be collapsed, we should probably just make it shorter instead.